### PR TITLE
chore: switch from cosmos-sdk/network to testnode package

### PR DESCRIPTION
**Pull request by cmwaters**
Closes: https://github.com/celestiaorg/celestia-app/issues/829

This refactors the three tests we have that use the CLI commands: `mint`, `blob` and `blobstream` to use the `testnode` package instead of `cosmos-sdk`'s `network` package.

The motivating reason for this is that cosmos-sdk's network package doesn't support initialising the `ConsensusParams` in the genesis file which we need to set the app version correctly

### CODE REVIEW

File: x/upgrade/legacy_test.go [hunk 1]
```diff
@@ -69,12 +69,9 @@ func (s *LegacyUpgradeTestSuite) SetupSuite() {
 		accounts[i] = tmrand.Str(9)
 	}
 
-	tmCfg := testnode.DefaultTendermintConfig()
-	tmCfg.Consensus.TimeoutCommit = 3 * time.Second
-
 	cfg := testnode.DefaultConfig().
 		WithFundedAccounts(accounts...).
-		WithTendermintConfig(tmCfg).
+		WithTendermintConfig(testnode.DefaultTendermintConfig()).
```
Comment (rootulp) [inline x/upgrade/legacy_test.go:74]: can this line be deleted entirely given testnode.DefaultConfig() already uses the default tendermint config?

https://github.com/celestiaorg/celestia-app/blob/047c38b5a7048a404dbc864aeb40e9b81b7f591e/test/util/testnode/config.go#L124

```suggestion
```

**Review by rootulp — DISMISSED** [review]
LGTM! Thanks for the clean up

**Review by staheri14 — APPROVED** [review]
Looks good!

File: x/mint/types/constants.go [hunk 2]
```diff
@@ -28,12 +31,18 @@ var (
 	initialInflationRateAsDec = sdk.NewDecWithPrec(InitialInflationRate*1000, 3)
 	disinflationRateAsDec     = sdk.NewDecWithPrec(DisinflationRate*1000, 3)
 	targetInflationRateAsDec  = sdk.NewDecWithPrec(TargetInflationRate*1000, 3)
+
+	initialInflationRateAsLegacyDec = math.LegacyNewDecWithPrec(InitialInflationRate*1000, 3)
```
[the remark this answers is not in the document]
Comment (cmwaters) [reply x/mint/types/constants.go:35]: Yeah I was going to

**Review by rootulp — APPROVED** [review]

**Review by evan-forbes — APPROVED** [review]
nice! thanks for doing this!

**Pull request merged**